gaffer-as-a-service/gaas-eureka-server/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
```

Comment:
You have ignored the jar but it is added with this PR.

Reply:
Has been deleted from PR